models/public/netvlad/netvlad.md

```diff
@@ -0,0 +1,94 @@
+# netvlad
+
+## Use Case and High-Level Description
+
+NetVLAD is a CNN architecture which tackles the problem of large scale visual place recognition. The architecture uses VGG 16 as base network and NetVLAD - a new trainable generalized VLAD (Vector of Locally Aggregated Descriptors) layer. It is a place recognition model pretrained on the [Pittsburgh](http://www.ok.ctrl.titech.ac.jp/~torii/project/repttile/) dataset.
```

Comment:
if I right understand, dataset name is Pittsburgh 250k

> Datasets
Place recognition datasets:
Tokyo Time Machine: available on request. The train/val splits are provided with our code.
Tokyo 24/7: available on request here.
Pittsburgh 250k: available on request here. The train/val/test splits are provided with our code.
Tiny subset of Tokyo Time Machine (21 MB). Contains 360 images, just to be used to validate if the NetVLAD code is set up correctly.

Reply:
Updated